build.override.targets

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0"  xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--if ServiceUri is set, the service has been already started on another machine-->
+  <Target Name="WcfSetup" Condition="($(WithCategories.Contains('OuterLoop')) or '$(OuterLoop)' == 'true') and ('$(ServiceUri)' == '') and $(OS.Contains('Windows'))" BeforeTargets="BuildAllProjects">
+    <Exec Command="$(MSBuildThisFileDirectory)\src\System.Private.ServiceModel\tools\scripts\WCFCustomSetup.cmd" ConsoleToMSBuild="true">
+      <Output TaskParameter="ConsoleOutput" PropertyName="WcfSetupOutput" />
+    </Exec>
+    <Message Text="WcfSetupOutput: $(WcfSetupOutput)" />
+  </Target>
+
```

Comment:
What I need is to run setup before test running and clean up after test has run.

I have tried BeforeTargets=Test for the setup target,  however the tests got run before setup was called. So I'm out of idea what the same target I could use. Could you suggest one?